betelgeuse.py

```diff
@@ -28,7 +39,40 @@ def parse_requirement_name(test_case_id):
     return parts[index].replace('test_', '').replace('_', ' ').title()
 
 
-@click.command()
+def parse_junit(path):
+    """Parse a jUnit XML file.
+
+    :param str path: Path to the jUnit XML file.
+    :return: A list of dicts with information about every test
+        case result.
+    """
+    root = ElementTree.parse(path).getroot()
+    result = []
+    for testcase in root.iter('testcase'):
+        data = testcase.attrib
+        element = testcase.find('error')
+        if element is not None:
```

Comment:
a lot of this code is repeated... can't we adopt a `for status in ['error', 'failure', 'skipped']` and avoid the repetition?